src/SFA.DAS.EmployerApprenticeshipsService.Web.UnitTests/StatusCalculator/WhenEditStatusIsWithBoth.cs

```diff
@@ -1,23 +0,0 @@
-﻿using FluentAssertions;
```

Comment:
The tests in these 3 unit test files were either duplicated in or have been rolled into WhenGettingStatusOfNonTransferCommitment.